src/middleware/requestValidator/middleware.ts

```diff
@@ -0,0 +1,30 @@
+import type { NextFunction, Request, Response } from "express";
+import {
+  checkSchema,
+  type Location,
+  validationResult,
+  type Schema,
+} from "express-validator";
```

Comment:
Selected https://express-validator.github.io/docs for now.
The way it is implemented in our code (middleware) makes it easy to change in the future if we are not happy with it. 
Also, even though `express-validator` provides ready to use middleware functions, I decided to create a custom one because it is easier to control the error response when the validation fails. Otherwise we would have to create a new middleware function to catch the specific error thrown by the library.

Existing other options:
- https://github.com/simonplend/express-json-validator-middleware (provides middleware function)
- https://github.com/ajv-validator/ajv (would have to be inserted in middleware function)